packages/store/src/helpers.js

```diff
@@ -0,0 +1,20 @@
+// @ts-check
+
+import { getInterfaceOf } from '@agoric/marshal';
+
+/**
+ * Helper function to reject keys which are empty objects but not marked as
+ * Remotable. This is intended to catch code which uses harden({}) (which
+ * will become pass-by-copy, see #2018) as a "handle" or "marker object"
+ * when they should have used Far().
+ *
+ * @param { unknown } key
+ */
+export function isEmptyNonRemotableObject(key) {
+  return (
+    typeof key === 'object' &&
+    Reflect.ownKeys(key).length === 0 &&
```

Comment:
`yarn lint` is happy with the following change

```suggestion
    typeof key === 'object' &&
    key !== null &&
    Reflect.ownKeys(key).length === 0 &&
```